### PR TITLE
Fluid clean move dygraph profiler, fluid.input.on_hot and fluid.input.embedding

### PR DESCRIPTION
**Pull request by GGBond8488**
### PR types
Others 

### PR changes
APIs

### Describe
remove no used dygraph.profiler and fluid.input.one_hot , move fluid.input.embedding to paddle.static.nn

### CODE REVIEW

File: python/paddle/static/nn/common.py [hunk 1]
```diff
@@ -3631,3 +3631,164 @@ def layer_norm(
...
+        param_attr(ParamAttr): To specify the weight parameter property. Default: None, which means the
+            default weight parameter property is used. In addition,
+            user-defined or pre-trained word vectors can be loaded with the :attr:`param_attr` parameter.
+            The local word vector needs to be transformed into numpy format, and the shape of local word
+            vector should be consistent with :attr:`size` .
+        dtype(str): It refers to the data type of output Tensor.
+            It must be float32 or float64. Default: float32.
+
+    Returns:
+        Variable: Embedding Tensor or LoDTensor mapped by input. The data type is the same as :attr:`dtype` .
```
Comment (zoooo0820) [inline python/paddle/static/nn/common.py]: 文档部分需要更新一下，例如`Varible`-> `Tensor`，重点检查一下概念是否符合2.0. 以及不出现lod相关概念

**Review by zoooo0820 — APPROVED** [review]
LGTM

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Comment by jeff41404** [comment]
replace input with one_hot and embedding  in the title?

**Comment by GGBond8488** [comment]
> replace input with one_hot and embedding in the title?

Indicate the API for specific cleaning in the title

**Review by jeff41404 — APPROVED** [review]
LGTM

**Review by lanxianghit — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Pull request merged**